src/java/org/apache/cassandra/transport/messages/QueryMessage.java

```diff
@@ -113,6 +114,13 @@ protected boolean isTrackable()
 
             QueryHandler queryHandler = ClientState.getCQLQueryHandler();
             statement = queryHandler.parse(query, state, options);
+            if (statement instanceof AlterSchemaStatement){
+                logger.info(String.format("DDL Query: %s, Keyspace: %s, by User: %s, From: %s",
+                                          query,
+                                          ((AlterSchemaStatement) statement).keyspace(),
+                                          state.getClientState().getUser().getName(),
+                                          state.getClientAddress()));
```

Comment:
now using the QueryState getClientAddress which if it's an internal query it would return null else the client address.  this looks to be better here than before directly with ClientState